checkers/aiService.ts

```diff
@@ -108,7 +108,7 @@ function getStateValue(board: Board): number {
 /**
  * Get the state score for player 0, a simple wrapper function
  */
-function getStateScoreForIndex0(state: IState): number {
+export function getStateScoreForIndex0(state: IState): number {
```

Comment:
don't make any checkers changes in your sound PR...

Reply:
I dont remember I changed it.. there is no reason to do it. I will delete it.